solutions/project/hub-env/fortigate/fortigate-ap-primary.yaml

```diff
@@ -0,0 +1,445 @@
+          # session-pickup has impact on cpu and may be disabled to improve performance
+          set session-pickup enable
+          set ha-mgmt-status enable
+          config ha-mgmt-interfaces
+              edit 1
+                  set interface "port4"
+                  set gateway 172.31.202.1
+              next
+          end
+          set override enable
+          set priority 200
+          set unicast-hb enable
+          set unicast-hb-peerip 172.31.202.11
+          set unicast-hb-netmask 255.255.255.0
+      end
+      config system sdn-connector
+          edit "gcp"
+              set type gcp
+              set ha-status enable
+          next
+      end
+      config system dns
+        set primary 169.254.169.254
+        set protocol cleartext
+        unset secondary
+      end
+      # Everything underneath this line will be synchronised to the secondary node with HA
+      # Explicit proxy for APPRZ and DATARZ workloads
+      config system settings
+          set gui-explicit-proxy enable
+      end
+      config web-proxy explicit
+          set status enable
+          set http-incoming-port 8080
+          set https-incoming-port 8080
+      end
+      # Internal Load balancers health check
+      ## VIP
+      config firewall vip
+            edit "ilb-healthcheck-vip"
+              set extip 172.31.201.30
+              set mappedip "169.254.255.100"
+              set extintf "port2"
+              set portforward enable
+              set extport 8008
+              set mappedport 8008
+          next
+          edit "ilb-proxy-healthcheck-vip"
+              set extip 172.31.201.35
+              set mappedip "169.254.255.100"
+              set extintf "port2"
+              set portforward enable
+              set extport 8008
+              set mappedport 8008
+          next
+      end
+      ## VIP Group
+      config firewall vipgrp
+          edit "ilb-healthcheck-vipgrp"
+              set interface "port2"
+              set member "ilb-healthcheck-vip" "ilb-proxy-healthcheck-vip"
+              set comment "This group contains VIP objects representing internal load balancers health checks. It is referenced in a policy forwarding traffic to the probe loopback interface"
+          next
+      end
+      ## Service
+      config firewall service custom
+          edit "PROBE"
+              set tcp-portrange 8008
+          next
+      end
+      ## Policy
+      config firewall policy
+          edit 0
+              set name "ilb healthcheck"
+              set srcintf "port2"
+              set dstintf "probe"
+              set action accept
+              set srcaddr "all"
+              set dstaddr "ilb-healthcheck-vipgrp"
+              set schedule "always"
+              set service "PROBE"
+              set comment "This policy forwards internal load balancers health checks to the probe loopback interface"
+          next
+      end
+---
+# # Instance Template
```

Comment:
If this commented code will not be used then I would remove it. If we plan to use it in the future perhaps we can store this some where else temporarily.

Reply:
removed in new commit